services/upload/src/main/resources/application.yaml

```diff
@@ -33,7 +33,7 @@ services:
       enable-dsos: ${ENABLE_DSOS_TRANSMISSION_TO_EFGS:true}
       enable-report-type: ${ENABLE_REPORT_TYPE_TRANSMISSION_TO_EFGS:true}
       default-dsos: ${DEFAULT_DSOS:1}
-      default-report-type: ${DEFAULT_REPORT_TYPE:0}
+      default-report-type: ${DEFAULT_REPORT_TYPE:UNKNOWN}
```

Comment:
when is this value used? only in case ENABLE_REPORT_TYPE_TRANSMISSION_TO_EFGS == false, right?

Reply:
Correct. This snippet shows how it's used:
```java
public ReportType getReportTypeOrDefault(ReportType type) {
    return canSendReportType ? type : defaultReportType;
}
```
This method is always called when we are converting the keys to be uploaded on a batch.